### PR TITLE
Astro doc init

### PR DESCRIPTION
**Pull request by wsxrdv**
Empty autogenerated documentation page

### CODE REVIEW

**Comment by jmichelp** [comment]
We probably want to exclude json, js, and ts files from spellchecking.

File: website/src/content/docs/index.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Welcome to Starlight
```
Comment (jmichelp) [inline website/src/content/docs/index.mdx:2]: I'm a bit puzzled. Some pages refer to SCAAML, others seem to be default/example ones from the Astro sometimes embedding resources such as a webp file that we don't need.

File: website/src/assets/houston.webp
Comment (jmichelp) [inline website/src/assets/houston.webp:1]: I think we could remove this file.

File: website/src/content/docs/index.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Welcome to SCAAML
+description: Side Channel Attacks Assisted with Machine Learning.
+template: splash
+hero:
+  tagline: Congrats on setting up a new Starlight project!
```
Comment (jmichelp) [inline website/src/content/docs/index.mdx:6]: Change line 6
Remove line 7-8

**Review by jmichelp — APPROVED** [review]
Just 2 minor changes.

**Review by kralka — APPROVED** [review]

**Pull request merged**